conf/azurebatch.config

```diff
@@ -0,0 +1,27 @@
+//Nextflow config file for running on Azure batch
+params {
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Venkat Malladi (@vsmalladi)'
+  config_profile_url = 'https://azure.microsoft.com/services/batch/'
+
+  az_location = 'westus2'
+  batch_name = false
+  batch_key = false
+  storage_name = 'platinumgenomes'
+  sas_token = 'sv=2019-02-02&se=2050-01-01T08%3A00%3A00Z&si=prod&sr=c&sig=FFfZ0QaDcnEPQmWsshtpoYOjbzd4jtwIWeK%2Fc4i9MqM%3D'
```

Comment:
I wonder if it's better to leave this as `false`. If you want the example then that's probably better in the readme. I can imagine some fairly confusing error messages if people leave this in..

Reply:
i tried false for both but got these issues:

`Missing AZURE_STORAGE_ACCOUNT_NAME`

Trying again now